tooling/lsp/src/requests/completion/tests.rs

```diff
@@ -1573,4 +1573,29 @@ mod completion_tests {
         "#;
         assert_completion(src, vec![field_completion_item("some_property", "i32")]).await;
     }
+
+    #[test]
+    async fn test_auto_import_with_super() {
+        let src = r#"
+            pub fn barbaz() {}
+
+            mod tests {
+                fn foo() {
+                    barb>|<
+                }
+            }
+        "#;
+        let items = get_completions(src).await;
+        assert_eq!(items.len(), 1);
+
+        let item = &items[0];
+        assert_eq!(item.label, "barbaz()");
+        assert_eq!(
+            item.label_details,
+            Some(CompletionItemLabelDetails {
+                detail: Some("(use super::barbaz)".to_string()),
+                description: Some("fn()".to_string())
+            })
```

Comment:
```suggestion
            pub fn bar_baz() {}

            mod tests {
                fn foo() {
                    bar_b>|<
                }
            }
        "#;
        let items = get_completions(src).await;
        assert_eq!(items.len(), 1);

        let item = &items[0];
        assert_eq!(item.label, "bar_baz()");
        assert_eq!(
            item.label_details,
            Some(CompletionItemLabelDetails {
                detail: Some("(use super::bar_baz)".to_string()),
                description: Some("fn()".to_string())
            })
```

Something to make spellcheck happy.